modules/document_repository/php/document_repository.class.inc

```diff
@@ -166,6 +165,9 @@ class Document_Repository extends \NDB_Menu_Filter
             $fileTypes[$fileType['File_type']] = $fileType['File_type'];
         }
         $fileTypes = array_merge(array(null => 'Any') + $fileTypes);
+        if (!isset($fileCategories)) {
```

Comment:
I don't think `$fileCategories` is set in any scenario at this point. Could we pass an empty array to `$this->addSelect('File_category', 'File category:', $fileCategories);` on line 177 ?